src/DonationForms/Actions/ConvertDonationFormBlocksToFieldsApi.php

```diff
@@ -131,6 +132,13 @@ protected function createNodeFromBlockWithUniqueAttributes(BlockModel $block, in
             case "givewp/donor-name":
                 return $this->createNodeFromDonorNameBlock($block);
 
+            //TODO: give_is_donor_comment_field_enabled() prevents this from showing up in admin
+            case "givewp/donor-comments":
+                return Textarea::make('comment')
+                    ->label($block->getAttribute('label'))
+                    ->helpText($block->getAttribute('description'))
+                    ->rules('max:255');
```

Comment:
Is this an existing rule? Are comments currently restricted to 255 characters?

Reply:
I noticed we were setting this rule on a few text fields which I think is because of the mysql data type - although now that i'm looking at it - that's for the meta_key not the value.  this one is stored in meta which says is longtext so I don't think we would ever reach that limit.

<img width="496" alt="Screenshot 2023-08-10 at 2 58 42 PM" src="https://github.com/impress-org/givewp-next-gen/assets/10138447/13ccf4c9-948e-4f8b-982d-ad7c1d13340a">

Reply:
Right, longtext has a monstrously large limit as it's an external file. I'd put a limit of 5,000 or something as a sanity limit, but that's about it.

Reply:
We only need to be setting a 255 limit for columns that are varchar.

Reply:
sounds good, resolved 6eddbce6